.github/workflows/ci.yml

```diff
@@ -65,7 +66,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        image: ["debian:testing", "debian:unstable"]
+        image: ["debian:buster", "debian:bullseye", "debian:testing", "debian:unstable"]
```

Comment:
It seem better to me to use 'oldstable', 'stable', 'testing' and 'unstable' instead of 'buster', 'bullseye', 'testing', 'unstable', to dynamically change when Debian make new releases.  No idea if it work, but recommend testing those names if you have not done so already.

Reply:
Admittedly, that change of names that we cannot control is why I prefer to have the distributions named explicitly. So, I would also s/testing/bookworm/.